src/main/java/seedu/cakecollate/logic/parser/CliSyntax.java

```diff
@@ -13,5 +13,5 @@
     public static final Prefix PREFIX_ORDER_DESCRIPTION = new Prefix("o/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_DATE = new Prefix("d/");
-
+    public static final Prefix PREFIX_REQUEST = new Prefix("r/");
```

Comment:
Request only takes in the request field right? So does it need a prefix? Because it will always be r/ and never anything else.

Reply:
i was wondering why the ab3 tutorial added a prefix too, i think its used to mark the start of the request, but im not too sure why they added it in the first place, so i left it as it is.

Reply:
ohh icic!